cli/lsp/mod.rs

```diff
@@ -1,5 +1,8 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
 
+// FIXME(bartlomieju): remove this attribute
+#![allow(unused)]
```

Comment:
There are 47 fields in `cli/lsp` (mostly in `cli/lsp/tsc.rs`) that are never read. This is caused by the fact that fields only used in `Debug` implementation of struct are no longer marked as used. I'd like to address it in a follow up PR.

CC @dsherret @kitsonk

Reply:
Actually these structs are used via JSON serialization.